src/vuex/modules/intercom.js

```diff
@@ -0,0 +1,34 @@
+import config from "src/../config"
+import { Intercom } from "capacitor-intercom"
+
+let intercom = null
+/* istanbul ignore next */
+if (config.mobileApp) {
+  intercom = new Intercom()
+  const userId = "lunie-app-" + Math.floor(Math.random() * 10000 + 1).toString()
+  intercom.registerIdentifiedUser({ userId })
+}
+
+export default () => {
+  return {
+    state: {
+      intercom
+    },
+    mutations: {
+      displayMessenger(state) {
+        state.intercom.displayMessenger()
+      },
+      displayHelpCenter(state) {
+        state.intercom.displayHelpCenter()
+      }
+    },
+    actions: {
+      displayMessenger({ commit }) {
```

Comment:
I guess we only need mutations or actions for this ^^ I guess only actions

Reply:
It put the mutations there cause I think not having it was causing problems with `index.spec.js` test, but that wasn't the cause of the problems :-).

I needed to modify transformIgnorePatterns in jest config to whitelist `capacitor-intercom` package, the problem is that this modification causes the tests to run very slow (>5min to complete). I want to figure out how to fix that.

```
transformIgnorePatterns: ["/node_modules/?!(capacitor-intercom)"],
```

Reply:
Have you considered mocking the `capacitor-intercom` module with `jest.mock` where it is imported instead of ignoring the transform?

Reply:
Thanks, good point!

Reply:
This is done and great! testing back to normal speed ;-)